iOS/classes/BleConnectionEvent.swift

```diff
@@ -0,0 +1,17 @@
+import Foundation
```

Comment:
Events are currently specified in `iOS/classes/BleEvent.swift`, please add them there for consistency.